framework/CodeInterfaces/PHISICS/phisicsdata.py

```diff
@@ -0,0 +1,961 @@
+"""
+Created on July 25th, 2017
+@author: rouxpn
+"""
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+import os
+import re
+import csv
+import xml.etree.ElementTree as ET
+from decimal import Decimal
+
+class phisicsdata():
+  """
+    This class parses the PHISICS output of interest. The output of interest are printed in the a csv file.
+  """
+  def __init__(self, phisicsDataDict):
+    """
+      Reads the PHISICS output.
+      @ In, phisicsDataDict, dictionary, dictionary of variables passed by the interface
+      @ Out, None
+    """
+    self.phisicsRelap = phisicsDataDict['phiRel']               # boolean: True means PHISICS RELAP coupled, False means PHISICS standalone
+    self.printSpatialRR = phisicsDataDict['printSpatialRR']     # boolean: True means spatial RR are printed, False means spatial RR are not printed
+    self.printSpatialFlux = phisicsDataDict['printSpatialFlux'] # boolean: True means spatial fluxes are printed, False means spatial fluxes are not printed
+    self.workingDir = phisicsDataDict['workingDir']             # string: working directory
+    self.mrtauOutputFileMPI = []
+    instantDict = {}
+    mrtauDict = {}
+    markerList = ['Fission matrices of','Scattering matrices of','Multigroup solver ended!']
+
+    self.instantOutputFileMPI,self.mrtauOutputFileMPI = self.fileOutName(phisicsDataDict)
+    mrtauOutput = [self.mrtauOutputFileMPI,phisicsDataDict['mrtauFileNameDict']['decay_heat']]
```

Comment:
burn up?

Reply:
What about burnup?

Reply:
you read the decay heat...but what about the burnup?